data/emission_factors.csv

```diff
@@ -1,100 +1,307 @@
-87,transport,plane,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Flight, long-haul, Premium economy class passenger",P.km,,,,long-haul,kerosine,kg/P.km,0.23659,premium_economy_class,with RF > 3700 km
-88,transport,plane,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Flight, long-haul, Business class passenger",P.km,,,,long-haul,kerosine,kg/P.km,0.42882,business_class,with RF > 3700 km
-89,transport,plane,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Flight, long-haul, First class passenger",P.km,,,,long-haul,kerosine,kg/P.km,0.59147,first_class,with RF > 3700 km
-90,transport,ferry,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Ferry, Foot passenger",P.km,,,,,,kg/P.km,0.01874,foot_passenger,with RF > 3700 km
-91,transport,ferry,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Ferry, Car passenger",P.km,,,,,,kg/P.km,0.12952,car_passenger,with RF > 3700 km
-92,transport,ferry,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Ferry, Average",P.km,,,,,,kg/P.km,0.11286,average,with RF > 3700 km
-93,transport,motorbike,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Motorbike, small",P.km,small,,,,,kg/P.km,0.08306,,
-94,transport,motorbike,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Motorbike, medium",P.km,medium,,,,,kg/P.km,0.1009,,
-95,transport,motorbike,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Motorbike, large",P.km,large,,,,,kg/P.km,0.13245,,
-96,transport,motorbike,"UK, Department for Business, Energy & Industrial Strategy",2020 UK GHG Conversion factors,"Motorbike, average",P.km,average,,,,,kg/P.km,0.11355,,
-97,transport,bicycle,UBA,,Fahrrad,P.km,,,,,,kg/P.km,0.009,,
-98,transport,pedelec,UBA,,Pedelec,P.km,,,,,,kg/P.km,0.015,,
+,country_code,category,subcategory,source,name,unit,size_class,range,fuel_type,co2e_unit,co2e,seating
+0,global,heating,,"UK, Department for Business, Energy & Industrial Strategy",coal (domestic),kWh,,,coal,kg/kWh,0.35,
+1,global,heating,,"UK, Department for Business, Energy & Industrial Strategy",natural gas,kWh,,,gas,kg/kWh,0.18,
+2,global,heating,,"UK, Department for Business, Energy & Industrial Strategy",LPG,kWh,,,liquid_gas,kg/kWh,0.21,
+3,global,heating,,"UK, Department for Business, Energy & Industrial Strategy",fuel oil,kWh,,,oil,kg/kWh,0.27,
+4,global,heating,,"UK, Department for Business, Energy & Industrial Strategy",wood pellets,kWh,,,pellet,kg/kWh,0.01074,
+5,global,heating,,"UK, Department for Business, Energy & Industrial Strategy",wood chips,kWh,,,woodchips,kg/kWh,0.01074,
+6,AF,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.1241,

```

Comment:
Nice with the country-specific electricity emission factors - we have to adapt the `calc_co2_electricity` function accordingly to take a country code as well and to have useful defaults :)

Reply:
done